Sources/Kafka/KafkaMetadata.swift

```diff
@@ -5,6 +5,7 @@ public struct KafkaMetadata: Sendable {
 
     init(metadata: UnsafePointer<rd_kafka_metadata>) {
         self.topics = (0..<Int(metadata.pointee.topic_cnt)).map { KafkaTopicMetadata(topic: metadata.pointee.topics[$0]) }
+        rd_kafka_metadata_destroy(metadata)
```

Comment:
Such ownership transfer from the caller looks weird...
Is there any possibility to destroy kafka metadata by the caller?

Reply:
Well, at least we have alloc/destroy in the same scope here...
Another possible approach would be to wrap the `UnsafePointer<rd_kafka_metadata>` into class box, and accept such box type as an argument for `KafkaMetadata`.